tests/ltp/umount02.patch

```diff
@@ -0,0 +1,81 @@
+diff --git a/testcases/kernel/syscalls/umount/umount02.c b/testcases/kernel/syscalls/umount/umount02.c
+index cd67aa6d2..8822eab89 100644
+--- a/testcases/kernel/syscalls/umount/umount02.c
++++ b/testcases/kernel/syscalls/umount/umount02.c
+@@ -20,6 +20,11 @@
+ #include "tst_test.h"
+ 
+ #define MNTPOINT	"mntpoint"
++const char* src  = "/dev/vda";
++const char* trgt = MNTPOINT; 
++const char* type = "ext4";
++const unsigned long mntflags = 0;
++const char* opts = "mode=0777";  
+ 
+ static char long_path[PATH_MAX + 2];
+ static int mount_flag;
+@@ -31,7 +36,7 @@ static struct tcase {
+ 	int exp_errno;
+ } tcases[] = {
+ 	{"Already mounted/busy", MNTPOINT, EBUSY},
+-	{"Invalid address", NULL, EFAULT},
++/*	{"Invalid address", NULL, EFAULT}, This test is commented because in SGX accessing NULL results in enclave exit */
+ 	{"Directory not found", "nonexistent", ENOENT},
+ 	{"Invalid  device", "./", EINVAL},
+ 	{"Pathname too long", long_path, ENAMETOOLONG}
+@@ -44,26 +49,42 @@ static void verify_umount(unsigned int n)
+ 	TEST(umount(tc->mntpoint));
+ 
+ 	if (TST_RET != -1) {
+-		tst_res(TFAIL, "umount() succeeds unexpectedly");
++		tst_res(TFAIL, "umount() succeeds unexpectedly. TEST_FAILED");
+ 		return;
+ 	}
+ 
+ 	if (tc->exp_errno != TST_ERR) {
+-		tst_res(TFAIL | TTERRNO, "umount() should fail with %s",
++		tst_res(TFAIL | TTERRNO, "umount() should fail with %s. TEST_FAILED",
```

Comment:
Why is this needed? Is the exit code of the test not enough?

Reply:
This was for a pipe line script to find result as per Huseyn.

Reply:
@hukoyu Can you comment?